packages/patrol_cli/lib/src/commands/develop.dart

```diff
@@ -110,6 +114,13 @@ class DevelopCommand extends PatrolCommand {
 
     final devices = await _deviceFinder.find(stringsArg('device'));
     final device = devices.single;
+
+    // `flutter logs` doesn't work on macOS, so we don't support it for now
+    // https://github.com/flutter/flutter/issues/138974
```

Comment:
I think there should be an issue on our repo, so we can track it. The Flutter issue should be linked to from our issue.